src/models/swap/basicSwap.ts

```diff
@@ -0,0 +1,75 @@
+import { Contract } from '@ethersproject/contracts'
+import Provider from '@services/provider'
+import getContract from '@utils/getContract'
+import { abi as ROUTERV2_ABI } from '@constants/abis/IUniswapV2Router02.json'
+import BaseSwap from './baseSwap'
+import { ROUTER_ADDRESS, BIPS_BASE } from '@constants/index'
+import { Currency, CurrencyAmount, JSBI, Percent, Router, Trade } from '@fuseio/fuse-swap-sdk'
+
+export default class BasicSwap extends BaseSwap {
```

Comment:
Can you make another PR renaming this class to FuseSwap?
`default class BasicSwap extends BaseSwap`  kinda weird